src/audio/kpb.c

```diff
@@ -1315,6 +1318,62 @@ static enum task_state kpb_draining_task(void *arg)
 	return SOF_TASK_STATE_COMPLETED;
 }
 
+/**
+ * \brief Copy s16 data linear to circular
+ *
+ * \param[in] source - source buffer
+ * \param[in] ostream - output stream
+ * \param[in] bytes_total - number of bytes to copy
+ *
+ * \return none.
+ */
+static inline void drain_s16(int16_t *source, struct audio_stream *ostream, size_t bytes_total)
```

Comment:
what is the reason for inlining?

Reply:
It's just for speed. It's used from one code location only. Also compiler might inline it but this makes it explicit. Now I wonder if these should be in header instead?